lighthouse-core/fraggle-rock/config/filters.js

```diff
@@ -48,16 +50,32 @@ function filterCategoriesByAvailableAudits(categories, availableAudits) {
   if (!categories) return categories;
 
   const availableAuditIds = new Set(availableAudits.map(audit => audit.implementation.meta.id));
+  const manualAuditIds = new Set(
```

Comment:
nit: Would a single `auditIdToAuditMeta` map also work? I don't think you need to change anything unless you see us adding another filter in the future.

Reply:
sure I'll refactor 👍